example/verifyartifact/main.go

```diff
@@ -102,7 +102,7 @@ func main() {
 		log.Fatal(err)
 	}
 
-	var b *bundle.Bundle
+	var b *bundle.ProtobufBundle
```

Comment:
@dnwe - why did you make this change?
I'm now getting a deprecation warning:

<img width="1294" alt="deprecation-2025-01-09_10-26-23" src="https://github.com/user-attachments/assets/af124d98-908f-4b8d-93a6-8c228989fd6f" />

Reply:
@gmlewis sigstore-go was rolled back to 0.5.1 in the sample, because in 0.6.2 they had similarly made a mistake of bumping the go directive to latest 1.23.2 (see [here](https://github.com/sigstore/sigstore-go/blob/e5df725510b9523147a78653d7b81fa5d0e9c7d1/go.mod#L3-L5)). They're addressing that according to [this comment](https://github.com/sigstore/fulcio/issues/1899#issuecomment-2572116404), but in the meantime we had to rollback in the sample in order to pin to Go 1.22. In 0.5.1 Bundle doesn't exist yet and ProtobufBundle isn't deprecated https://github.com/sigstore/sigstore-go/blob/v0.5.1/pkg/bundle/bundle.go#L51-L55